src/main/starlark/builtins_bzl/common/python/providers.bzl

```diff
@@ -96,6 +99,11 @@ the same conventions as the standard CPython interpreter.
             "script used when executing `py_binary` targets.  Does not " +
             "apply to Windows."
         ),
+        "bootstrap_template": (
+            "The bootstrap script template file to use. Should have %python_binary%, " +
+            "%workspace_name%, %main%, and %imports%. See " +
+            "@bazel_tools//tools/python:python_bootstrap_template.txt for more variables."
+        ),
```

Comment:
nit: Lets just direct the reader to the py_runtime.bootstrap_template docs instead.